met-web/src/components/dashboard/EngagementAccordion.tsx

```diff
@@ -0,0 +1,88 @@
+import React, { useState } from 'react';
+import Grid from '@mui/material/Grid';
+import { MetBody, MetLabel } from 'components/common';
+import { Accordion, AccordionDetails, AccordionSummary } from '@mui/material';
+import { formatDate } from 'components/common/dateHelper';
+import { AppConfig } from 'config';
+import { Engagement } from 'models/engagement';
+import { When } from 'react-if';
+
+const EngagementsAccordion = ({
```

Comment:
very minor, this is named EngagementsAccordion and the file is named EngagementAccordion.